csm_web/frontend/static/frontend/css/policies-styles.css

```diff
@@ -11,14 +11,10 @@
 
 .policy-sections h1 {
 	color: var(--csm-green);
-	/*
-		CSM green, but IDK if I can import vars from other CSS files lol.
-		OG color was: #646464 (feel free to change back, just trying out green!)
-	*/
 }
 
 .policy-sections a {
-	color: var(--csm-link-blue);
```

Comment:
Wait this was right the first time lol

Reply:
oops yeah lol no autosave like in intellij oops